blockchain/blockchain-wallet-v4-frontend
# Kyc new designs modal

## Description
- Added new modal GetStarted for KyC
- Added cartridge New in left menu next to Exchange
- Added new entry in Whatsnew menu
- Added banner on the homepage for KyC when modal is not displayed anymore
- Updated GetStarted screen to know reflect the KyC status instead of the previous banner.
- Removed Shapeshift Exchange journey
- Removed SFOX Timeline banner

## Change Type
Please enter one or more of the following: 
- Feature
- Upgrades

## Testing Steps

For both desktop and mobile resolutions with new wallet:
-> Check if Modal is displayed the first time
-> After logging in again, the modal should not be there anymore, but a banner on the homepage instead
-> Check the new entry in whatsnew works correctly
-> Check that the GetStarted screen reflect the right status and copy (see here different versions:
https://projects.invisionapp.com/share/NDOR4H3WF5C#/screens/329502776)
-> Check at each GetStarted status that the profile reflects the same KyC status

## Code Checklist
